ga4gh/datamodel/variants.py

```diff
@@ -423,6 +440,11 @@ def getVariants(self, referenceName, startPosition, endPosition,
         # protocol version 0.6
         if callSetIds is None:
             callSetIds = []
+        else:
+            for call in callSetIds:
+                if call not in self._callSetIds:
+                    raise exceptions.CallSetNotInVariantSetException(
```

Comment:
You're right, sorry I should have looked this up. We still need to change the parameters though: it's expecting callSetId and variantSetId.